docs/plugins/testing_workshop_docs/2-pytest-testing-frameworks.md

```diff
@@ -0,0 +1,116 @@
+# 2: Pytest testing framework  
+
+This lesson explains how to use the pytest testing framework to make testing easier.
```

Comment:
Because it is the first time we are mentioning it, I suggest adding a link to the pytest official page here:

```suggestion
This lesson explains how to use the [pytest testing framework](https://docs.pytest.org/en/7.2.x/) to make testing easier.
```